jetbrains-core/resources/META-INF/ext-rider.xml

```diff
@@ -30,9 +30,14 @@
     </extensions>
 
     <actions>
-        <group>
+        <group id="aws.toolkit.serverless.template.rider">
             <add-to-group group-id="SolutionExplorerPopupMenu" anchor="last"/>
-            <reference ref="aws.toolkit.serverless.template"/>
+            <add-to-group group-id="EditorPopupMenu" anchor="last"/>
+            <add-to-group group-id="EditorTabPopupMenu" anchor="last"/>
+
+            <separator/>
+            <action id="serverless.application.deploy.rider" class="software.aws.toolkits.jetbrains.services.lambda.actions.RiderDeployServerlessApplicationAction"/>
```

Comment:
Do we need to do an `<unregister>`?

Reply:
Done. Unregister original deploy action in Rider action group to avoid check for Rider IDE.